docs/es/week03/03-1.md

```diff
@@ -330,26 +329,25 @@ The right side shows the ventral pathway, which tells you what you're looking at
 
 <center><img src="{{site.baseurl}}/images/week03/03-1/Simon_Thorpe.png" alt="Simon_Thorpe" style="zoom:55%;" /></center>
 
-<div align="center">Fig. 11. Modelo de flujo de información visual al cerebro de Simon Thorpe <div>
-
-Las señales pasen de la retina al LGN (este ayuda con la mejora del contraste, el  manejo de compuertas, etc.), luego proceden al córtex visual primario V1, V2, V4,  seguido del córtex inferotemporal (PIT), el cual, es la parte del cerebro dónde las categorías están definidas. Observaciones de cirugías cerebrales han mostrado que, si a un humano le es enseñado un filme, las neuronas en el PIT se activaran únicamente cuando detecten ciertas imágenes -- como Jennifer Aniston o la abuelita de alguna persona -- y de ninguna otra forma. Las respuestas neuronales son invariantes a cualidades como posición, tamaño, iluminación, la orientación de la abuelita, o cómo está vestida esta.
+<div align="center">Fig. 11. Modelo de Simon Thorpe de flujo de información visual en el cerebro</div>
 
+Las señales pasan de la retina al núcleo geniculado lateral (LGN, por sus siglas en inglés), que ayuda con la mejora del contraste, el  manejo de compuertas, etc. Luego proceden al córtex visual primario V1, V2, V4,  seguido del córtex inferotemporal (PIT, por sus siglas en inglés), el cual es la parte del cerebro donde las categorías están definidas. Observaciones de cirugías abiertas cerebrales han mostrado que si se le muestra a un humano una película, las neuronas en el PIT se activarán únicamente cuando detecten ciertas imágenes&mdash;como Jennifer Aniston o la abuela de alguna persona&mdash;y de ninguna otra forma. Las respuestas neuronales son invariantes a cualidades como posición, tamaño, iluminación, la orientación de la abuela, o cómo está vestida ella, etc.
 
-Asimismo, la rapidez del tiempo de reacción con que los humanos puede clasificar estos objectos --apenas suficiente para que algunos picos pasen-- demuestra que es posible realizar esta tarea sin el tiempo adicional requerido para operaciones recurrentes más complejas. Más bien, esto es un proceso de un pase hacia adelante (single feed-forward) solamente.
+Asimismo, la rapidez del tiempo de reacción con la que los humanos fueron capaces de clasificar estos objectos&mdash;apenas tiempo suficiente para que algunos picos pasen&mdash;demuestra que es posible realizar esta tarea sin el tiempo adicional requerido para operaciones recurrentes más complejas. Más bien, esto es un proceso de un solo pase hacia adelante (*single feed-forward*).

```

Comment:
error: change "&mdash" to "--"

Reply:
I changed them back, although it is not an error; it is the way to render a proper em dash on Markdown (it's their HTML code).